torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -201,7 +205,8 @@ void ProfilingRecord::insertShapeProfile(Node* n, size_t offset) {
 }
 
 bool needsProfiledInputs(Node* n) {
-  if (tensorexpr::isSupported(n)) {
+  if (tensorexpr::isSupported(n) ||
+      (RegisterCudaFuseGraph::isRegistered() && fuser::cuda::canFuseNode(n))) {
```

Comment:
Calls to `fuser::` needs to be guarded by `!C10_MOBILE`
```suggestion
  if (tensorexpr::isSupported(n) ||
#ifndef C10_MOBILE
      (RegisterCudaFuseGraph::isRegistered() && fuser::cuda::canFuseNode(n))
#else
      false
#endif
     ) {
```